src/web/vaev-layout/base.cpp

```diff
@@ -457,6 +458,36 @@ export struct Input {
     }
 };
 
+// https://drafts.csswg.org/css-align-3/#baseline-set
+// https://drafts.csswg.org/css-writing-modes-3/#baseline
+// https://www.w3.org/TR/css-inline-3/#baseline-types
+// https://www.w3.org/TR/css-inline-3/#dominant-baseline-property
+// NOTE: positions are relative to box top, not absolute
+export struct BaselinePositionsSet {
```

Comment:
discuss with CLVB if we really need a new struct for this
pros:
- Au and not f64
- relative position to Box top and not baseline
cons:
- more complexity